lms/static/js/instructor_dashboard/membership.js

```diff
@@ -966,11 +995,28 @@ such that the value can be defined later than this assignment (file load order).
                 authList = $opt.data('auth_list');
                 authList.$container.addClass('active');
                 authList.re_view();
+
+                // On Change update the Group Moderation list
+                if ($opt.text() === 'Discussion Group Community TA') {
```

Comment:
Should this be using .text() instead of the option's value? Just wondering if using the .text will cause i18n issues.